### ISSUE TITLE
Missing current version of lanchain

### ISSUE THREAD

**Issue by alexprice12** [hwchase17/langchain#1009]
When I try to install with `pip install` for the current version I get the following error:

```
ERROR: Could not find a version that satisfies the requirement langchain==0.0.84 (from versions: 0.0.1, 0.0.2, 0.0.3, 0.0.4, 0.0.5, 0.0.6, 0.0.7, 0.0.8, 0.0.9, 0.0.10, 0.0.11, 0.0.12, 0.0.13, 0.0.14, 0.0.15, 0.0.16, 0.0.17, 0.0.18, 0.0.19, 0.0.20, 0.0.21, 0.0.22, 0.0.23, 0.0.24, 0.0.25, 0.0.26, 0.0.27)
ERROR: No matching distribution found for langchain==0.0.84
```

**Comment by alexprice12** [comment]
I was using the wrong version of python. The package is available on versions of python greater than 3.8

**Comment by daieven** [comment]
please executer this command in terminal and try again later ,  "pip install --upgrade pip "
